unit_tests/test_redis_client_base.py

```diff
@@ -0,0 +1,235 @@
+"""
```

Comment:
Why is this part of this PR? Isnt' it targeted in https://github.com/gridsingularity/d3a-api-client/pull/163 ?
I would suggest to re-open a  new PR that only contains the new changes. Thanks!